packages/react-vapor/src/components/editor/CodeEditorActions.ts

```diff
@@ -0,0 +1,21 @@
+import {IReduxAction} from '../../utils';
+
+export const CodeEditorActionTypes = {
+    add: 'ADD_CODE_EDITOR',
+    remove: 'REMOVE_CODE_EDITOR',
```

Comment:
`add` and `remove` don't seem to be used, but they probably should (especially `remove` to remove the code editor entry from the store).

Reply:
:+1:  good point I'll add an action to clear the code editor from the store and call this action in the end of life cycle.